services/traction/requirements-bdd.txt

```diff
@@ -0,0 +1,78 @@
+aiohttp==3.7.4.post0
```

Comment:
👍 👍 keeping this separate from the traction service requirements!

Reply:
there is probably alot more that can be removed, but i'm not sure, so i'll revisit trimming that file down later